config/cscs.py

```diff
@@ -1129,14 +1126,23 @@
         {
             'name': 'PrgEnv-nvidia',
             'target_systems': [
-                'dom', 'daint'
+                'daint'
             ],
             'modules': [
                 'PrgEnv-nvidia',
                 # FIXME: We should not be forcing a cdt version
                 'cdt/21.05'
             ]
         },
+        {
+            'name': 'PrgEnv-nvidia',
+            'target_systems': [
+                'dom'
+            ],
+            'modules': [
+                'PrgEnv-nvidia',
+            ]
+        },
```

Comment:
The only reason that these were so sparsely populated was because they were auto-generated from the old config. We should not follow that scheme and become more compact when adding new things or editing existing entries.

Reply:
Do you mean changing the format to put arrays on one-line

Reply:
Yes, I can do that for this entry.